### PR TITLE
Content for requests to Safe ecosystem foundation.md

### LINKED ISSUE #50: Bounty proposal: Content for requests to Safe ecosystem foundation
Previously it was agreed that I will compile the content for requests to the Safe ecosystem foundation to apply here
https://notionforms.io/forms/get-in-touch-with-the-safe-ecosystem-foundation

Topics for requests
1. Safe to provide bounties for DevPool
2. Ubiquity to create a Dapp to the Safe marketplace
- The grant topic is discussed separately, so it should not be included in these topics.

Questions
- COMPANY/DAO/PROJECT NAME
- COMPANY/PROJECT WEBSITE 
- PLEASE TELL US IN DETAIL WHAT YOUR PROJECT DOES, WHAT YOU WANT TO DO WITH THE SAFE, AND WHERE THE CORE CONTRIBUTORS CAN HELP (150 CHARACTERS MINIMUM) 

The initial discussion https://github.com/orgs/ubiquity/projects/24/views/4?pane=issue&itemId=31314735

### PR DESCRIPTION
**Pull request by sergfeldman**
Resolves #50

### CODE REVIEW

File: Content for requests to Safe ecosystem foundation.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+### 1. Safe to provide bounties for DevPool
+
+**COMPANY/DAO/PROJECT NAME**
+
+DevPool by Ubiquity DAO
+
+**COMPANY/PROJECT WEBSITE** 
+
+https://github.com/ubiquity/devpool-directory
```
Comment (0x4007) [inline Content for requests to Safe ecosystem foundation.md]:
```suggestion
https://github.com/ubiquity/devpool-directory/issues
```

File: Content for requests to Safe ecosystem foundation.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Ubiquity DAO uses Gnosis Chain for payouts to DAO members and freelancers.  
+
+We offer Gnosis and Safe Ecosystem to partner in the creation of the Safest crypto card.
+
+Users' funds will be protected with the complete set of security features:
+- Using the Safe crypto wallet
+- Shielding the card information with temporary virtual cards (as on https://privacy.com/)
+- Ability to configure card transaction multisig with on-chain and common 2FA methods (a customizable analog of 3DS)
+- Abilitity to configure white- and black-lists, as well as spending limits for transactions (by MCC, countries, specific destinations)
+- Ability to protect funds by temporary card freezing in case or customizable fraud triggers
```
Comment (0x4007) [inline Content for requests to Safe ecosystem foundation.md]: You're speaking as if we already have the card available. Perhaps its more accurate to state that it'll be available soon™️

**Review by 0x4007 — CHANGES_REQUESTED** [review]
Mostly good just think that it should be made clear that the card isn't available. I also think that maybe we can think of tighter integrations with the Gnosis infrastructure. 

Lastly this makes me think that we should update our index ubq.fi copywriting at the top.

File: Content for requests to Safe ecosystem foundation.md (hunk 2 above)
Comment (sergfeldman) [reply Content for requests to Safe ecosystem foundation.md]: Thank you, updated to highlight that the card is planned

File: Content for requests to Safe ecosystem foundation.md (hunk 1 above)
Comment (sergfeldman) [reply Content for requests to Safe ecosystem foundation.md]: @pavlovcik I accepted this suggestion but then realised that this point is about COMPANY/PROJECT WEBSITE and I reverted the link.

The link to DevPool issues is provided as well in other point - PLEASE TELL US IN DETAIL WHAT YOUR PROJECT DOES ...
https://github.com/ubiquity/devpool-directory/issues

Maybe change the link in the point COMPANY/PROJECT WEBSITE to https://dao.ubq.fi/devpool ?

File: Content for requests to Safe ecosystem foundation.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+https://github.com/ubiquity/devpool-directory/
+
+**PLEASE TELL US IN DETAIL WHAT YOUR PROJECT DOES, WHAT YOU WANT TO DO WITH THE SAFE, AND WHERE THE CORE CONTRIBUTORS CAN HELP (150 CHARACTERS MINIMUM)** 
+
+DevPool is a Web3 freelance marketplace on top of GitHub. 
+It scales development throughput with talents available on GitHub and automates routine freelance management activities.
+Ubiquity uses Gnosis Chain for payouts to DAO members and freelancers.
+
+DevPool is in beta testing with the open issues listed here https://github.com/ubiquity/devpool-directory/issues
+Currently, the cost for DevPool services is only payout for freelancers without any fees for the DevPool itself.
```
Comment (0x4007) [inline Content for requests to Safe ecosystem foundation.md]:
```suggestion
Currently, the cost for DevPool services is for projects to pay freelancers. However we do not incur any fees at this time.
```

File: Content for requests to Safe ecosystem foundation.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+We offer that DevPool will solve several issues currently open in your GitHub projects https://github.com/safe-global https://github.com/gnosischain
```
Comment (0x4007) [inline Content for requests to Safe ecosystem foundation.md]:
```suggestion
We know that using our system will grant Gnosis DAO the capability to directly financially incentivize the community to close out open issues on https://github.com/safe-global https://github.com/gnosischain
```

File: Content for requests to Safe ecosystem foundation.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+- Ubiquity uses Gnosis Chain for payouts to DAO core contributors and bounty hunters.  
+
+We offer Gnosis and Safe Ecosystem to partner in the creation of the Safest crypto card. 
+
+It is planned that users' funds will be protected with the complete set of security features:
+- Using SAFE.
+- Shielding card information with temporary virtual cards (as on https://privacy.com/).
+- Ability to configure card transaction multisig with on-chain and common 2FA methods (a customizable analog of 3DS).
+- Ability to configure white and blacklists, as well as spending limits for transactions (by MCC, countries, specific destinations).
+- Ability to protect funds by temporary card freezing in case of customizable fraud triggers.
```
Comment (0x4007) [inline Content for requests to Safe ecosystem foundation.md:68]: All the card related stuff seems a bit irrelevant to Gnosis I would imagine. Perhaps it's better to exclude those details, and maybe even the stablecoin. 

I say this because they aren't dependencies of the DevPool offering necessarily. It obfuscates our proposal from having them focus on the most important offering: using the bot.

**Review by 0x4007 — CHANGES_REQUESTED** [review]

File: Content for requests to Safe ecosystem foundation.md (hunk 5 above)
Comment (Hodlatoor) [reply Content for requests to Safe ecosystem foundation.md:68]: agreed. less is better and more focused with them at this point.

**Comment by Hodlatoor** [comment]
@sergfeldman did you remove the card and other items @pavlovcik mentioned? I agree that a more streamlined, narrower scope is likely better at this point.

**Comment by sergfeldman** [comment]
> @sergfeldman did you remove the card and other items @pavlovcik mentioned? I agree that a more streamlined, narrower scope is likely better at this point.

@pavlovcik @Hodlatoor I accepted all the comments for this PR.

The current status is defined in the issue comment
https://github.com/ubiquity/business-development/issues/50#issuecomment-1640701333
